tests/rustdoc/deref/deref-methods-24686-target.rs

```diff
@@ -0,0 +1,27 @@
+#![crate_name = "foo"]
+
+// test for https://github.com/rust-lang/rust/issues/24686
+use std::ops::Deref;
+
+pub struct Foo<T>(T);
+impl Foo<i32> {
+    pub fn get_i32(&self) -> i32 { self.0 }
+}
+impl Foo<u32> {
+    pub fn get_u32(&self) -> u32 { self.0 }
+}
+
+// Note that the same href is used both on the method itself,
```

Comment:
Isn't it an issue?

Reply:
no, since we're testing that it appears twice, and in any case, we care about the negative case much more, which will trigger if the bad method shows up in the sidebar or on the main page.